### PR TITLE
changes

### PR DESCRIPTION
**Pull request by Mattches1Kn**

### CODE REVIEW

**Comment by Mattches1Kn** [comment]
Please review this, @casey-collab

File: german/Schlafen/schlafen.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Glückwunsch! 
+
+Dazu …
+
+
+… benutzen Sie den [solarbetriebenen Motor](solarbetriebener-Motor/solarbetriebener-Motor.md).
```
Comment (casey-collab) [inline german/Schlafen/schlafen.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: german/ruderboot.md [hunk 2]
```diff
@@ -17,3 +17,5 @@ Werden Sie:
 [Aus dem Traum aufwachen und Oktoberfest besuchen](Oktoberfest/oktoberfest.md)
 
 [Sie beschliessen fortan keinen Alkohol mehr zu trinken.](Alkohol/alkohol.md)
+
+[Weiterschlafen.](Schalfen/schlafen.md)
```
Comment (casey-collab) [inline german/ruderboot.md]: This link is also invalid.

File: german/Schlafen/schlafen.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+Glückwunsch!
```
Comment (casey-collab) [inline german/Schlafen/schlafen.md]: It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by Mattches1Kn** [comment]
Please review this, @casey-collab

File: german/Schlafen/schlafen.md (hunk 1 above)
Comment (casey-collab) [inline german/Schlafen/schlafen.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
Closing this pull request due to lack of activity. If you ever have time to respond to the comments above, feel free to re-open it.

**Pull request closed without merging**